features/raadpleeg-bewoning-met-periode/test/bewoners-volgorde.feature

```diff
@@ -0,0 +1,459 @@
+      | adresseerbaarObjectIdentificatie |          0800010000000001 |
+    En heeft de bewoning bewoners met de volgende gegevens
+      | burgerservicenummer | naam.voornamen | naam.geslachtsnaam | geboorte.datum |
+      |           000000024 | Karel          | Jansen             |       20011014 |
+      |           000000012 | Femke          | Pietersen          |       19980526 |
+      |           000000036 | Elise          | Boer               |       19971130 |
+      |           000000061 | Irene          | Jansen             |       20000328 |
+      |           000000048 | Karel Jan      | Jansen             |       19990624 |
+      |           000000085 | Bas            | Pietersen          |       20000119 |
+      |           000000073 | Bas            | Pietersen          |       20190917 |
+      |           000000115 | Ruud           | Alberts            |       20160703 |
+    En heeft de bewoning mogelijke bewoners met de volgende gegevens
+      | burgerservicenummer | naam.voornamen | naam.geslachtsnaam | geboorte.datum |
+      |           000000103 | Leonie         | Teunissen          |       20181103 |
+      |           000000139 | Peter          | Teunissen          |       20210411 |
+      |           000000097 | Corine         | Alberts            |       20171109 |
+      |           000000140 | Anton          | Boer               |       20140621 |
+    En heeft de response een bewoning met de volgende gegevens
+      | naam                             | waarde                    |
+      | periode                          | 2024-01-01 tot 2024-05-01 |
+      | adresseerbaarObjectIdentificatie |          0800010000000001 |
+    En heeft de bewoning bewoners met de volgende gegevens
+      | burgerservicenummer | naam.voornamen | naam.geslachtsnaam | geboorte.datum |
+      |           000000024 | Karel          | Jansen             |       20011014 |
+      |           000000012 | Femke          | Pietersen          |       19980526 |
+      |           000000036 | Elise          | Boer               |       19971130 |
+      |           000000061 | Irene          | Jansen             |       20000328 |
+      |           000000048 | Karel Jan      | Jansen             |       19990624 |
+      |           000000103 | Leonie         | Teunissen          |       20181103 |
+      |           000000139 | Peter          | Teunissen          |       20210411 |
+      |           000000085 | Bas            | Pietersen          |       20000119 |
+      |           000000073 | Bas            | Pietersen          |       20190917 |
+      |           000000115 | Ruud           | Alberts            |       20160703 |
+      |           000000097 | Corine         | Alberts            |       20171109 |
+      |           000000140 | Anton          | Boer               |       20140621 |
+
+  @valideer-volgorde
+  Scenario: Sorteren mogelijke bewoners met gedeeltelijk onbekende datum aanvang volgende verblijfplaats
```

Comment:
waar ik mij zorgen om maak is dat ik scenarios zie waarbij ik niet begrijp welke business rules voor sortering worden toegepast, maar dat ik ook niet begrijp waarom sommige scenarios slagen terwijl het volgens mij niet klopt wat in de scenario staat beschreven. Bijv. in deze scenario woont volgens mij maar 1 persoon op adres 'A1' want alle andere personen staan ingeschreven op adres 'A2' en als de bewoning wordt gevraagd van adres 'A1' zie ik allemaal personen terug die op adres 'A2' wonen. Het zal wel aan mij liggen dat ik complexe scenarios niet kan lezen

Reply:
En wat ik mij ook steeds meer afvraag, is of het niet simpeler is om gewoon datum aanvang adreshouding te leveren, zodat consumers zelf kunnen bepalen hoe ze willen sorteren. Nu wordt er volgens mij best wel complexe logica toegevoegd die niets te maken hebben met de bewoning functionaliteit